www/js/con-student.js

```diff
@@ -266,7 +331,67 @@ define([
 				//this.applyDirectives(dd);
 			}
 		},
+		initialViewSettings: function(type){
+			//make display none for all fields initially
+			//removed optionality container from initial view settings
+			//TODO: further clean up necessary after discussion
+			if(type == "quantity"){
+				this.eqElements.forEach(function(elem){
+					console.log("element",elem);
+					style.set(elem,"display","none");
+				});
+				this.qtyElements.forEach(function(elem){
+					console.log("element",elem);
+					style.set(elem,"display","block");
+				});
+				/*
+				//can the qty node have delete option
+				var canHaveDeleteNode = this.canHaveDeleteNode();
+				if(canHaveDeleteNode){
+					registry.byId("deleteButton").set("disabled",false);
+				}
+				else{
+					registry.byId("deleteButton").set("disabled",true);
+				}
+				*/
+			}else if(type == "equation"){
+				this.qtyElements.forEach(function(elem){
+					console.log("element",elem);
+					style.set(elem,"display","none");
+				});
+				this.eqElements.forEach(function(elem){
+					console.log("element",elem);
+					style.set(elem,"display","block");
+				});
+				//enable the deletebutton always
+				registry.byId("deleteButton").set("disabled",false);
+			}
+			//emptying message box when a node is open
+			console.log("emptying message");
+			dojo.empty("messageOutputbox");
+		},
 		initialControlSettings: function (nodeid) {
+			var nodeType = this._model.student.getType(nodeid);
+			if(nodeType == "equation"){
+				var descriptionWidget = registry.byId(this.controlMap.description);
+				var equationWidget = registry.byId(this.controlMap.equation);
+				//for the schema select load schema options
+				this.loadSchemaOptions();
+				//for the entity select load entities
+				var entityWid = registry.byId(this.controlMap.entity);
+				var entityList = [];
+				var authorEntities = this._model.getAllEntities();
+				authorEntities.sort();
+				array.forEach(authorEntities, function(ent){
+					var obj = {value: ent, label: ent};
+					entityWid.addOption(obj);
+				}, this);
+
+				//disable the description and equation fields
+				descriptionWidget.set('disabled', true);
+				equationWidget.set('disabled', true);	
+			}
+			/*
```

Comment:
@riteshsamala  Is the plan to uncomment this after we get the new controls in?  The description part is obsolete but quantity node variable name, type, and other fields should all work the same as before...